crates/rolldown_plugin_transform/src/lib.rs

```diff
@@ -58,16 +59,21 @@ impl Plugin for TransformPlugin {
         return Err(anyhow::format_err!("Error occurred when parsing {}\n: {:?}", args.id, errs));
       }
     };
+
+    let env = if self.target.is_some() && self.browserslist.is_some() {
+      return Err(anyhow::anyhow!(
+        "Cannot specify both `target` and `browserslist` at the same time"
+      ));
+    } else if let Some(target) = &self.target {
+      EnvOptions::from_target(target).expect("Failed to create transform options")
```

Comment:
These should return the error messages from these functions, instead of panicking with `expect`.